### PR TITLE
WIP: add es6 suffix to lint

### PR DESCRIPTION
**Pull request by doits**
My javascript files have an `.es6` suffix. I want them to be linted. For `eslint` there is [no way to specify the suffix in the config file](https://github.com/eslint/eslint/blob/41db67020602809291dcd2458ac14a8a4b7d3d0d/docs/user-guide/configuring.md#specifying-file-extensions-to-lint). Besides this, `.js` is hardcoded in `pronto-eslint` anyway.

What can we do about it? I think hardcoding suffix is not the best way. Maybe there should be a possibility to configure `pronto-eslint` with the suffixes at least until `eslint` allows to specify file suffixes in the regular config?

This PR atm simply adds `.es6` as a suffix, but I think it should be configurable in some way. Do you have an idea how to do it?

### CODE REVIEW

**Comment by mmozuras** [comment]
@doits I'm 🆗 with hardcoding here. I don't think that there are many custom suffixes people use for es files.

**Comment by doits** [comment]
@mmozuras I'm fine with hardcoding `.es6` too, since then all my files are linted and I need no more ;-)

But I think there are a lot of different suffixes out there for javascript files (for example I read someone using `.jsx` in some eslint issues), but maybe it would be over engineered to implement it as a configuration option atm (maybe eslint will have support to specify the suffixes in it's own config at some time, the `pronto-eslint` will simply use this).

I added some specs (in a very rudimentary way), so from my side you can merge this and release a new version :-)

File: spec/fixtures/test.git/git/COMMIT_EDITMSG [hunk 1]
```diff
@@ -0,0 +1,7 @@
+add es6 file
```
Comment (mmozuras) [inline spec/fixtures/test.git/git/COMMIT_EDITMSG]: Could you remove this file 😄.  It's unnecessary.

**Comment by mmozuras** [comment]
@doits cool, other than my one comment, looks good 😄

**Comment by doits** [comment]
@mmozuras done

**Comment by mmozuras** [comment]
@doits thanks, released as 0.6.1 👏

**Pull request merged**